udata/core/dataservices/models.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+from udata.api_fields import field, generate_fields
+from udata.core.badges.models import BadgeMixin
+from udata.core.dataset.models import Dataset
+from udata.core.metrics.models import WithMetrics
+from udata.i18n import lazy_gettext as _
+import udata.core.contact_point.api_fields as contact_api_fields
+import udata.core.dataset.api_fields as datasets_api_fields
+
+from udata.models import db
+
+# "frequency"
+# "harvest"
+# "internal"
+# "page"
+# "quality" # Peut-être pas dans une v1 car la qualité sera probablement calculé différemment
+# "datasets" # objet : liste de datasets liés à une API
+# "spatial"
+# "temporal_coverage"
+
+DATASERVICE_FORMATS = ['REST', 'WMS', 'WSL']
```

Comment:
We should probably add it in `settings`. I'm unsure about all possible dataservice formats, but maybe we should add this when making harvesting work to get a cleared idea?

Reply:
Yes, it may be better to wait to have some examples.